identity-service/src/notifications/index.js

```diff
@@ -109,14 +109,14 @@ class NotificationProcessor {
     // Indexes network notifications
     this.notifQueue.process(async (job, done) => {
       let error = null
-
       let minBlock = job.data.minBlock
-      if (!minBlock && minBlock !== 0) throw new Error('no min block')
-
-      // Re-enable for development as needed
-      // this.emailQueue.add({ type: 'unreadEmailJob' })
 
       try {
+        if (!minBlock && minBlock !== 0) throw new Error('no min block')
```

Comment:
This same type of check is done in the Solana notifications queue like 50 lines below this. Should that also be updated for consistency?

Reply:
absolutely! pushed @Kyle-Shanks